nixos/modules/programs/wayland/sway.nix

```
@@ -45,7 +45,9 @@ in {
     enableRealtime = mkEnableOption (lib.mdDoc ''
       add CAP_SYS_NICE capability on `sway` binary for realtime scheduling
       privileges. This may improve latency and reduce stuttering, specially in
-      high load scenarios'') // { default = true; };
+      high load scenarios.
+      Enabling this option will allow any program run by `@users` group to
```

```suggestion
      Enabling this option will allow any program run by the "users" group to
```

@group is PAM-specific syntax, right?  That's an implementation detail.

---

Makes sense. Done.